### PR TITLE
Redis scaler example

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- Redis scaler example

### CODE REVIEW

**Review by Cortey — DISMISSED** [review]

File: examples/redis-scaler/Readme.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+## Scenario
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:1]: Please change the file name to README.md

File: examples/redis-scaler/Readme.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+This example realises scaling of jobs based on the size of [list](https://redis.io/docs/data-types/lists/) in redis.
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:3]:
```suggestion
This example realizes the scaling of jobs based on the size of [list](https://redis.io/docs/data-types/lists/) in Redis.
```

File: examples/redis-scaler/Readme.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+Each job, when scheduled, is poping a message from the list and stops.
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:4]:
```suggestion
When scheduled, each job pops a message from the list and stops.
```

File: examples/redis-scaler/Readme.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+Additionally there is a function that awaits  HTTP trafic and populates messages to redis list.
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:5]:
```suggestion
Additionally, there is a function that awaits  HTTP traffic and populates messages to the Redis list.
```

File: examples/redis-scaler/Readme.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+![diagram](assets/scenario.drawio.svg)
+
+## Prerequisistes
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:9]:
```suggestion
## Prerequisites
```

File: examples/redis-scaler/Readme.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+ - `KUBECONFIG` env variable pointing to a kubernetes instance
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:11]:
```suggestion
 - `KUBECONFIG` env variable pointing to a Kubernetes instance
```

File: examples/redis-scaler/Readme.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+ - helm client installed locally in your machine
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:12]:
```suggestion
 - Helm client installed locally on your machine
```

File: examples/redis-scaler/Readme.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+ - keda and serverless modules enabled in th kyma cluster (`make ensure_kyma_modules`)
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:13]:
```suggestion
 - The Keda and Serverless modules enabled in the kyma cluster (`make ensure_kyma_modules`)
```

File: examples/redis-scaler/Readme.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+
+## Deploy 
+
+### Deploy redis
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:18]:
```suggestion
1. Deploy Redis
```

File: examples/redis-scaler/Readme.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+
+Install redis into `redis` namespace.
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:20]:
```suggestion
Install Redis in the `redis` namespace.
```

File: examples/redis-scaler/Readme.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+```sh
+kubectl create ns redis
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo update
+helm install my-release bitnami/redis --namespace redis
+```
+
+### Deploy example
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:29]:
```suggestion
2. Deploy the example
```

File: examples/redis-scaler/Readme.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+ - keda and serverless modules enabled in th kyma cluster (`make ensure_kyma_modules`)
+
+
+## Deploy 
+
+### Deploy redis
+
+Install redis into `redis` namespace.
+
+```sh
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:22]:
```suggestion
   ```sh
```

File: examples/redis-scaler/Readme.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+kubectl create ns redis
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:23]:
```suggestion
   kubectl create ns redis
```

File: examples/redis-scaler/Readme.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+helm repo add bitnami https://charts.bitnami.com/bitnami
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:24]:
```suggestion
   helm repo add bitnami https://charts.bitnami.com/bitnami
```

File: examples/redis-scaler/Readme.md [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+helm repo update
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:25]:
```suggestion
   helm repo update
```

File: examples/redis-scaler/Readme.md [hunk 16]
```diff
@@ -0,0 +1,102 @@
...
+helm install my-release bitnami/redis --namespace redis
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:26]:
```suggestion
   helm install my-release bitnami/redis --namespace redis
```

File: examples/redis-scaler/Readme.md [hunk 17]
```diff
@@ -0,0 +1,102 @@
...
+```
+
+### Deploy example
+
+```sh
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:31]:
```suggestion
   ```sh
```

File: examples/redis-scaler/Readme.md [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+make deploy_app
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:32]:
```suggestion
   make deploy_app
```

File: examples/redis-scaler/Readme.md [hunk 19]
```diff
@@ -0,0 +1,102 @@
...
+### Deploy redis
+
+Install redis into `redis` namespace.
+
+```sh
+kubectl create ns redis
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo update
+helm install my-release bitnami/redis --namespace redis
+```
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:27]: Please add three spaces in line 27. When I try to add it as a suggestion, they disappear.

File: examples/redis-scaler/Readme.md [hunk 20]
```diff
@@ -0,0 +1,102 @@
...
+
+### Deploy example
+
+```sh
+make deploy_app
+```
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:33]: Please add three spaces in line 33. When I try to add it as a suggestion, they disappear.

File: examples/redis-scaler/Readme.md [hunk 21]
```diff
@@ -0,0 +1,102 @@
...
+kubectl port-forward -n redis svc/emitter-fn  8080:80
+```
+
+Send a few messages via POST request towards emitter-fn
+
+```sh
+curl -H "Content-Type: application/json" -X POST -d '{"msg":"hello1"}' localhost:8080
+curl -H "Content-Type: application/json" -X POST -d '{"msg":"hello2"}' localhost:8080
+curl -H "Content-Type: application/json" -X POST -d '{"msg":"hello3"}' localhost:8080
+```
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:66]: Please add three spaces in line 66. When I try to add it as a suggestion, they disappear.

File: examples/redis-scaler/Readme.md [hunk 22]
```diff
@@ -0,0 +1,102 @@
...
+processor-scaled-job-n68tp-x2p64    1/1     Running             0             2s
+processor-scaled-job-n68tp-x2p64    0/1     Completed           0             3s
+processor-scaled-job-vn4lg-g694x    0/1     Completed           0             3s
+processor-scaled-job-vn4lg-g694x    0/1     Completed           0             5s
+processor-scaled-job-n68tp-x2p64    0/1     Completed           0             5s
+processor-scaled-job-vn4lg-g694x    0/1     Completed           0             6s
+processor-scaled-job-n68tp-x2p64    0/1     Completed           0             6s
+```
+
+By inspecting logs of the jobs you should see all messages were processed
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:96]:
```suggestion
When you inspect the jobs logs, you should see that all messages have been processed.
```

File: examples/redis-scaler/Readme.md [hunk 23]
```diff
@@ -0,0 +1,102 @@
...
+```sh
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:97]:
```suggestion
   ```sh
```

File: examples/redis-scaler/Readme.md [hunk 24]
```diff
@@ -0,0 +1,102 @@
...
+kubectl logs -n redis -l scaledjob.keda.sh/name=processor-scaled-job -f
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:98]:
```suggestion
   kubectl logs -n redis -l scaledjob.keda.sh/name=processor-scaled-job -f
```

File: examples/redis-scaler/Readme.md [hunk 25]
```diff
@@ -0,0 +1,102 @@
...
+Processing started for hello2.. will finish in 8283ms
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:99]:
```suggestion
   Processing started for hello2.. will finish in 8283ms
```

File: examples/redis-scaler/Readme.md [hunk 26]
```diff
@@ -0,0 +1,102 @@
...
+Processing started for hello3.. will finish in 1523ms
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:100]:
```suggestion
   Processing started for hello3.. will finish in 1523ms
```

File: examples/redis-scaler/Readme.md [hunk 27]
```diff
@@ -0,0 +1,102 @@
...
+Processing started for hello1.. will finish in 8969ms
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:101]:
```suggestion
   Processing started for hello1.. will finish in 8969ms
```

File: examples/redis-scaler/Readme.md [hunk 28]
```diff
@@ -0,0 +1,102 @@
...
+```
```
Comment (grego952) [inline examples/redis-scaler/Readme.md:102]: Please add three spaces in line 102. When I try to add it as a suggestion, they disappear.

**Review by grego952 — APPROVED** [review]

**Review by Cortey — APPROVED** [review]

**Pull request merged**